scripts/zephyr_module.py

```diff
@@ -345,7 +345,8 @@ def kconfig_snippet(meta, path, kconfig_file=None, blobs=False, sysbuild=False):
 
 def process_kconfig(module, meta):
     blobs = process_blobs(module, meta)
-    taint_blobs = len(tuple(filter(lambda b: b['status'] != 'D', blobs))) != 0
+    blobs_found = tuple(filter(lambda b: b['status'] != BLOB_NOT_PRESENT, blobs))
+    taint_blobs = len(blobs_found) != 0
```

Comment:
nit: If you're looking for something more concise:
```suggestion
    taint_blobs = any(b['status'] != BLOB_NOT_PRESENT for b in blobs)
```

Reply:
Much sleeker, applied.
Thanks.